scripts/ci/install-helm.sh

```diff
@@ -1,30 +1,34 @@
 #!/bin/bash
 
 export HELM_INSTALL_DIR=/usr/bin
+HELM_VER="v2.9.1"
 HELM_GET=https://raw.githubusercontent.com/kubernetes/helm/master/scripts/get
+helm='helm --tiller-connection-timeout 10'
 
 uninstall() {
 	sudo rm -rf $HELM_INSTALL_DIR/helm
 }
 
 install() {
-	curl $HELM_GET | sed 's/helm version/helm --debug version/' | sh
+	sudo dnf install -y socat
```

Comment:
This is already done by our ansible playbooks

Reply:
ok, removed